idaes/surrogate/pysmo/sampling.py

```diff
@@ -720,29 +817,23 @@ def __init__(self, data_input, number_of_samples=None, sampling_type=None):
         print('Sampling type: ', self.sampling_type, '\n')
 
         if self.sampling_type == 'selection':
-            if isinstance(data_input, pd.DataFrame):
-                data = data_input.values
-                data_headers = data_input.columns.values.tolist()
-            elif isinstance(data_input, np.ndarray):
-                data = data_input
-                data_headers = []
+            if isinstance(data_input, (pd.DataFrame, np.ndarray)):
+                self.selection_columns_preprocessing(data_input, xlabels, ylabels)
             else:
-                raise ValueError('Pandas dataframe or numpy array required.')
-            self.data = data
-            self.data_headers = data_headers
+                raise ValueError('Pandas dataframe or numpy array required for sampling_type "selection."')
 
             # Catch potential errors in number_of_samples
             if number_of_samples is None:
                 print("\nNo entry for number of samples to be generated. The default value of 5 will be used.")
```

Comment:
print -> log (warning)

Reply:
For now PySMO uses print statements throughout the entire package. I'd like to keep it consistent for now.

Changing all the print statements in PySMO to logs will a separate undertaking and will be addressed in a separate PR later.